suite-native/module-onboarding/src/screens/OnboardingAssets.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { CryptoIcon, Icon } from '@trezor/icons';
+import TrezorConnect, { AccountInfo } from '@trezor/connect';
+import { Screen, StackProps } from '@suite-native/navigation';
+import {
+    Box,
+    Button,
+    Text,
+    Card,
+    InputWrapper,
+    Input,
+    SelectableListItem,
+    VStack,
+} from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { OnboardingStackParamList, OnboardingStackRoutes } from '../navigation/routes';
+import { AssetsLoader } from '../components/AssetsLoader';
+import { setOnboardingFinished } from '../slice';
+
+const assetsStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    padding: utils.spacings.medium,
+    justifyContent: 'space-between',
+}));
+
+export const OnboardingAssets = ({
+    route,
+}: StackProps<OnboardingStackParamList, OnboardingStackRoutes.OnboardingAssets>) => {
+    const dispatch = useDispatch();
+    const [radioChecked, setRadioChecked] = useState<string>('firstSelectable');
+    const [inputText, setInputText] = useState<string>('');
+    const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
+    const { applyStyle } = useNativeStyles();
+
+    const { xpubAddress, currencySymbol } = route.params;
+
+    const getAccountInfo = useCallback(() => {
+        TrezorConnect.getAccountInfo({
+            coin: currencySymbol,
+            descriptor: xpubAddress,
+        })
+            .then(accountInfo => {
+                if (accountInfo?.success) {
+                    setAccountInfo(accountInfo.payload);
+                }
+                // TODO error?
+            })
+            .catch(error => {
+                // eslint-disable-next-line no-console
+                console.log('getAccountInfo failed: ', JSON.stringify(error));
+            });
+    }, [xpubAddress, currencySymbol]);
```

Comment:
Why not use thunk and then store account info into store?

Reply:
Yes, let's definitely do it. But I would prefer to do it in the follow-up task I have created yesterday - https://github.com/trezor/trezor-suite/issues/5927. This PR and its related issue was about preparing screens for onboarding and it is quite big and mixing different things together right now. What do you think?

Reply:
Sure!